src/views/Feed/Feed.js

```diff
@@ -16,18 +16,18 @@ import FeedSummary from './FeedSummary'
 import FeedUser from './FeedUser'
 
 function Feed(props) {
-  const { children, className, items, size } = props
+  const { children, className, events, size } = props
   const classes = cx('ui', className, size, 'feed')
   const rest = getUnhandledProps(Feed, props)
 
-  if (!items) {
+  if (!events) {
     return <div {...rest} className={classes}>{children}</div>
   }
 
-  const eventsJSX = items.map(({ key, event }, index) => {
+  const eventsJSX = events.map(({ key, ...eventData }, index) => {
     const finalKey = key || index
```

Comment:
> Limit the props to strings only.

I don't understand how I can limit this. Proptypes validation?

------

Also, it will break doc example (see `Feed/Types/Props.js`). May be it might be simplified by more shorthand props.

----

In any case, it is better to make review before. I can't escape from feeling that I did something 	nasty there.